flow/workflows/cdc_flow.go

```diff
@@ -554,12 +546,6 @@ func CDCFlowWorkflow(
 
 		if shared.ShouldWorkflowContinueAsNew(ctx) {
 			restart = true
-			if syncFlowFuture != nil {
-				if err := model.SyncStopSignal.SignalChildWorkflow(ctx, syncFlowFuture, struct{}{}).Get(ctx, nil); err != nil {
```

Comment:
Have to decide whether cdc_flow restarts hard pause sync flow, otherwise have to setup a whole session state thing so workflow can signal activity to shutdown